### DISCUSSION TITLE
Multiple wildcards in one certificate on multiple DNS providers ?

### DISCUSSION THREAD

**Discussion by palijn** [acmesh-official/acme.sh#5167] [Q&A]
Hi,
I've got multiple wildcards in ONE certificate ( *.a.tld , *.b.tld, *.c.tld ).
At the time of issue, all domains were managed by the same DNS provider (1984.is).
Now one of the domains is managed by a different DNS provider (Cloudflare).
Is acme.sh capable of managing the renewal of all the wildcards in one certificate using multiple DNS providers ?
If yes, how should I proceed ?
Thanks a lot for your advices !

**Comment by neilpang** [comment]
https://github.com/acmesh-official/acme.sh/wiki/How-to-issue-a-cert#3-multiple-domains-san-mode--hybrid-mode
